version_check.sh

```diff
@@ -108,13 +108,19 @@ if [ "${product}" = "ps56" -o "${product}" = "ps57" -o "${product}" = "ps80" ];
     exit 1
   fi
 
-  if [ ${product} = "ps80" ]; then
-    if [ "$(mysqlsh --version | grep -c ${version})" = 1 ]; then
-      echo "mysql-shell version is correct" >> "${log}"
-    else
-      echo "ERROR: mysql-shell version is incorrect"
-      exit 1
+  if [ -z ${install_mysql_shell} ] || [ ${install_mysql_shell} = "yes" ] ; then
+    if [ ${product} = "ps80" ]; then
```

Comment:
Actually we want to check the mysql shell version using install_mysql_shell variable. 

So I kept install_mysql_shell as the main if condition after which it will further propagate and check for the product version.

I got it there might be issues because of it, if the variable install_mysql_shell is not defined properly then it will not bother to check other things like product=ps80.

Reply:
Discussed in slack.